src/components/Homepage/BodySection/Card/FeatureCard.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { CardProps } from '../../HomepagePageContent';
+import { Links } from '../../../ProductPage/BodySection/Card/Links';
+import { SvgObject } from '../../../SvgObject';
+
+export const FeatureCard = ({ title, content, image, links }: CardProps) => (
+  <div className="flex justify-between flex-col" style={{ maxWidth: '29.125rem' }}>
```

Comment:
I would get rid of this `maxWidth` attribute and let the parent element decide this. Also `HeroCard` & `SdkCard` doesn't have this restriction.